sdk/resourcemanager/azure-resourcemanager-appservice/CHANGELOG.md

```diff
@@ -6,11 +6,10 @@
 
 - Supported `withContainerSize` for `FunctionApp`.
 
-### Breaking Changes
-
 ### Bugs Fixed
 
-### Other Changes
+- Updated SKU that automatically set Function App "Always On".
+Function App on `FREE`, `SHARED`, `DYNAMIC` (consumption plan), `ELASTIC_PREMIUM` (premium plan), `ELASTIC_ISOLATED` App Service has "Always On" turned off.
```

Comment:
How do we decide which SKU to have "Always On" turned off?

Reply:
I were asking Functions service in email.

Currently this only one uncertain is `ELASTIC_ISOLATED`

> Free – Not Supported.
Basic, Dedicated, Premium App Service Plans (Not Elastic) – The value should be set. (Defaults to On)
Functions Premium Plan – The value should not be set.
Isolated\ASE – The Value should be set.